package.json

```diff
@@ -88,9 +88,9 @@
     "@graphql-codegen/add": "1.17.7",
     "@graphql-codegen/cli": "1.17.10",
     "@graphql-codegen/introspection": "1.18.0",
-    "@graphql-codegen/typescript": "1.17.10",
+    "@graphql-codegen/typescript": "1.15.1",
     "@graphql-codegen/typescript-graphql-files-modules": "1.18.0",
-    "@graphql-codegen/typescript-operations": "1.17.8",
+    "@graphql-codegen/typescript-operations": "1.15.1",
```

Comment:
We need to downgrade to this version to prevent the generation of the `type Exact`.
This behavior is then fixed in the upcoming v17 release #1677